### PR TITLE
Adding first draft of the contributing file

### PR DESCRIPTION
**Pull request by hab-anka**
# What changed, and why it matters

Added first draft of the contributing file.
Still need to:
- all links
- check and link references

### CODE REVIEW

File: CONTRIBUTING.rst
Comment (lornajane) [inline CONTRIBUTING.rst:11]: I've been writing this as "DevPortal" but I'm not sure we have an agreed standard anywhere.

File: CONTRIBUTING.rst
Comment (lornajane) [inline CONTRIBUTING.rst:18]: (add a blank line between the paragraph and the list to make the bullet points render correctly)

File: CONTRIBUTING.rst
Comment (lornajane) [inline CONTRIBUTING.rst:25]: Since this is public content, we should avoid referring to any internal documents. I think it would be OK to remove this item since people will either fit in with the existing structure, or we'll figure something out when we review their PR.

File: CONTRIBUTING.rst
Comment (lornajane) [inline CONTRIBUTING.rst:34]: I think we can only ask people to check the issues list since the article request backlog isn't publicly available. Could you also say what you're asking them to check for? (I'm assuming duplicate issues/requests, but I could be missing something else here)

File: CONTRIBUTING.rst
Comment (lornajane) [inline CONTRIBUTING.rst:51]: Is the content model something we'll include in this pull request? Or could we create a more bare bones `CONTRIBUTING` file to get us started and then revisit to add these resources when they're ready?

File: CONTRIBUTING.rst
Comment (lornajane) [inline CONTRIBUTING.rst:70]: Again, if these are internal resources, they can't be linked here. Probably we can skip the style guides for most contributors - but should write internal guidelines for maintainers/reviewers that could include these.

**Review by lornajane — COMMENTED** [review]
Just adding a comment as this is still a work in progress but I think it covers most of the main points. The formatting also needs checking (I can do that if you would like help) before this would be ready to merge.

File: CONTRIBUTING.rst
Comment (hab-anka) [reply CONTRIBUTING.rst:11]: One for the automatic check?

File: CONTRIBUTING.rst
Comment (hab-anka) [reply CONTRIBUTING.rst:51]: I added the content model as a section in this file - **What is the content model?**. It's just here for the ease of use. I think we can keep it as it is now. And then update the file with links to templates (TBD).

File: CONTRIBUTING.rst
Comment (hab-anka) [inline CONTRIBUTING.rst:67]: @lornajane what is the size you use as default? Should we standardize that?

File: CONTRIBUTING.rst
Comment (lornajane) [inline CONTRIBUTING.rst:29]: Could we link to the framework resources in case anyone is interested? I'll be linking back from that project to this one as well at some point.

File: CONTRIBUTING.rst
Comment (lornajane) [inline CONTRIBUTING.rst:117]: Templates should be in restructuredtext format so that people can use it as a basis for their contributions (edit: the same comment for all the templates)

File: CONTRIBUTING.rst
Comment (lornajane) [inline CONTRIBUTING.rst:96]: Typo, should be "ReStructuredText".  Should this also be a link to somewhere?

**Review by lornajane — CHANGES_REQUESTED** [review]
Missing a link (or two, there are probably a few rst resources we can share), and the templates should be ReStructuredText also so that people can copy and paste.

**Comment by hab-anka** [comment]
Included links, and reformatted the templates. @lornajane, have a look.

File: CONTRIBUTING.rst
Comment (diawarr) [inline CONTRIBUTING.rst:14]: It's >> Its

File: CONTRIBUTING.rst
Comment (diawarr) [inline CONTRIBUTING.rst:27]: peoplr >> people

File: CONTRIBUTING.rst
Comment (diawarr) [inline CONTRIBUTING.rst:92]: I wonder is it worth changing this to "Important" or something else? It might be more of a general documentation standard, but I'm use to rather strict categorization for warning, caution, and danger notifications, where warnings are usually reserved for situations that could result in serious physical injury or death :)

**Review by diawarr — COMMENTED** [review]
Just a couple of minor typos and a more general question about the use of warnings, but otherwise everything looks good :)

File: CONTRIBUTING.rst
Comment (hab-anka) [reply CONTRIBUTING.rst:92]: Agree!

**Comment by hab-anka** [comment]
Thanks for having a look @diawarr. Back to @lornajane for approval.

**Review by lornajane — APPROVED** [review]
Looking good, thanks for working on this

**Pull request merged**
